docs/dagster-university/pages/dagster-dbt/lesson-5/3-creating-assets-that-depend-on-dbt-models.md

```diff
@@ -107,53 +107,52 @@ Now we’re ready to create the asset!
    At this point, the `airport_trips` asset should look like this:
     
    ```python
-    @asset(
-        deps=["location_metrics"],
-    )
-    def airport_trips(database: DuckDBResource) -> MaterializeResult:
-        """
-            A chart of where trips from the airport go
-        """
-
-        query = """
-            select
-                zone,
-                destination_borough,
-                trips
-            from location_metrics
-            where from_airport
-        """
-
-        with database.get_connection() as conn:
-            airport_trips = conn.execute(query).fetch_df()
-
-        fig = px.bar(
-            airport_trips,
-            x="zone",
-            y="trips",
-            color="destination_borough",
-            barmode="relative",
-            labels={
-                "zone": "Zone",
-                "trips": "Number of Trips",
-                "destination_borough": "Destination Borough"
-            },
-        )
-
-        pio.write_image(fig, constants.AIRPORT_TRIPS_FILE_PATH)
-
-        with open(constants.AIRPORT_TRIPS_FILE_PATH, 'rb') as file:
-            image_data = file.read()
+   @asset(
```

Comment:
What is the difference in this block?

Reply:
the code block has a leading space which is a little tedious to copy over.

Reply:
ty ty